test/Microsoft.TestPlatform.AcceptanceTests/ProcessTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+using Microsoft.TestPlatform.TestUtilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.TestPlatform.AcceptanceTests;
+
+[TestClass]
+public class ProcessTests : AcceptanceTestBase
+{
+    [TestMethod]
+    [NetCoreTargetFrameworkDataSource]
+    public void MissingFrameworkTextIsCorrectlyDisplayed(RunnerInfo runnerInfo)
+    {
+        // Arrange
+        SetTestEnvironment(_testEnvironment, runnerInfo);
+        using var tempDir = new TempDirectory();
+        var assemblyPath = GetAssetFullPath("SimpleTestProjectMessedUpTargetFramework.dll", Core21TargetFramework);
+        var runtimeConfigJson = Path.Combine(Path.GetDirectoryName(assemblyPath), "SimpleTestProjectMessedUpTargetFramework.runtimeconfig.json");
+        var fileContent = File.ReadAllText(runtimeConfigJson);
+        var updatedContent = fileContent.Replace("\"version\": \"2.1.0\"", "\"version\": \"42\"");
+        File.WriteAllText(runtimeConfigJson, updatedContent);
```

Comment:
this test is modifying the assets right?

Reply:
Yes that's why I created a new test asset. I originally thought that we were doing a local copy of the asset for each test but we don't so I wasn't sure how to best do this.

Given the question, I should probably put a comment in the code to explain what/why.

Thanks for pointing it out!